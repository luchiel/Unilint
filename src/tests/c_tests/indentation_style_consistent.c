int b = -100;

int main() {
    int a = 0;
    while(a > b)
    	a--;
	printf("heresy");
}