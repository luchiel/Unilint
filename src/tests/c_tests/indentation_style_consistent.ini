[indentation]
indentation_style = consistent
