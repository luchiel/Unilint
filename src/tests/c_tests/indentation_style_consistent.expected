6:1: tabs and spaces mixed
7:1: tabs found, spaces expected
